src/__tests__/proposals.int.test.ts

```diff
@@ -124,6 +124,8 @@ describe('/proposals', () => {
     });
 
     it('returns a subset of proposals present in the database when search is provided', async () => {
+      // This should pass even if the default text search config is not english.
+      await db.query('set default_text_search_config = \'arabic\';');
```

Comment:
> It was pretty easy to add a single statement to the existing test, setting the default config to 'arabic', which then causes the test to fail if 'english' is not specified in the select query. Does this look OK?

I'd prefer to have regression tests split out into separate tests - setting the default text search config isn't really a part of this basic search test, and tests shouldn't be doing multiple things. In this case, there should be a test that search works at all, and a test that search works with a config other than `english` set.

I'd also prefer that the regression test use the `simple` config rather than `arabic`, since that's where we stumbled. Pair that with a search term that is stemmed in the `english` config (as the search term used in this test, `'summary'`, happens to be) but not in `simple` (since `simple` doesn't do stemming) and test that there is one search result.

Reply:
I separated out the test that uses `'simple'`. My reasoning behind arabic was it would be more likely to trigger the failure in the future, but I think you're right about the regression test covering exactly what we experienced.